encoding/protobuf/reflection/reflection.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package reflection
+
+// ServerReflectionInfo encapsulates the information about a service required
+// for exposing this service on the gRPC reflection protocol.
+// See https://github.com/grpc/grpc/blob/master/doc/server-reflection.md
+type ServerReflectionInfo struct {
```

Comment:
naming nit: we can drop `reflection` from the name since this is in the reflection package already.

Maybe something like `ServerMeta` so that it would be imported as `reflection.ServerMeta` (I suggest meta over info since we've used meta around the code base already)

Reply:
Looking at the codebase `ServerMeta` seems to indeed conform to most standards, I'll change it to that.